### ISSUE TITLE
Default sorting does not sort by backend

### ISSUE THREAD

**Issue by klaernie** [sni/Thruk#1181]
**Describe the bug**
The default sort order does not use the backend/site column as a default sort order.

**Thruk Version**
2.48

**To Reproduce**
1. visit the services page where identically named services are running on more than one backend
2. refresh the page 
3. observe the order of the services change

**Expected behavior**
The resulting list of service is sorted stable, so the first entry always points to the same backend.

**Screenshots**
the following two screenshots were made with the same service detail page, only refreshing the page between them.
![image](https://user-images.githubusercontent.com/5636383/172687992-8e98eb2b-29f7-4e6c-992d-2b586025c0ec.png)
![image](https://user-images.githubusercontent.com/5636383/172688008-3a985301-bd74-4408-965e-766578fece41.png)

**Desktop:**
 - Google Chrome 102.0.5005.63 (Official Build) (64-bit)

**Additional context**
On a technical level this is not a real problem, but it confuses end users.
I've also more than once triggered a recheck from an all-services-of-host page on one of the two backends, to have a before and after image for a ticket resolution, but then got confused, as the sort order changed and the check I was expecting to be listed first suddenly appeared below it's partner.

This problem is also not likely to hit most people, our environment is probably a bit ballistic by running the entire landscape of nagios servers twice for high availability, but then displaying everything in a single Thruk instance. But this makes it easy for operations to handle problems so that both sides of our HA clusters know about the ack, comment etc.

**Comment by klaernie** [comment]
okay, the problem is still reproduced with lmd 2.0.7, so I updated the initial text

**Comment by sni** [comment]
Indeed, the site name is not used for sorting because usually i would try to make either the hostname or the service description unique.
But anyway, since this is a rare, but possible, scenario, the sort order should not jump between refreshes.
